src/rtccertificate.cc

```diff
@@ -25,17 +25,27 @@ static const char kExpires[] = "expires";
 static const char kFingerprints[] = "fingerprints";
 static const char kAlgorithm[] = "algorithm";
 static const char kValue[] = "value";
+static const char kPrivateKey[] = "privateKey";
+static const char kCertificate[] = "certificate";
+static const char kFromPEM[] = "fromPEM";
+static const char kToPEM[] = "toPEM";
 
 NAN_MODULE_INIT(RTCCertificate::Init) {
   Local<FunctionTemplate> ctor = Nan::New<FunctionTemplate>(New);
   ctor->SetClassName(LOCAL_STRING(sRTCCertificate));
   ctor->InstanceTemplate()->SetInternalFieldCount(1);
 
+  Nan::SetMethod(ctor, kFromPEM, FromPEM);
```

Comment:
Well, what I meant was that `LOCAL_STRING(kFromPEM)` fails to compile, because `Nan::SetMethod` has a different signature than `Nan::SetAccessor`, leading to 
```
cannot convert ‘Nan::New(((const char*)(& kFromPEM))).v8::MaybeLocal<T>::ToLocalChecked<v8::String>()’ (type ‘v8::Local<v8::String>’) to type ‘const char*’
```
I think it's correct as-is.